src/layercake.js

```diff
@@ -121,11 +124,17 @@ export default class LayerCakeStore extends Store {
 	}
 
 	computeValues (settings, originalSettings) {
-		this.compute('padding', ['target', 'containerWidth', 'containerHeight'], target => {
+		this.compute('padding', ['target', 'containerWidth', 'containerHeight', 'ssr'], (target, containerWidth, containerHeight, ssr) => {
 			const defaultPadding = {top: 0, right: 0, bottom: 0, left: 0};
 			let hasPadding = false;
 			const padding = {};
 
+			if (ssr) {
+				// TODO: Can padding be defined in config if it's a computed
```

Comment:
You should be able to return the same `Object.assign(defaultPadding, settings.padding || {});` no?

Reply:
That makes sense to me.